pytorch/training/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -125,7 +130,6 @@ RUN mkdir /tmp/efa \
  && tar -xf aws-efa-installer-${EFA_VERSION}.tar.gz \
  && cd aws-efa-installer \
  && apt-get update \
- && apt-get upgrade -y \
```

Comment:
1. For the apex package, I'm passing the question to sm model parallel team. Additionally, `torch.cuda.amp` should be the path forward replacing `nvidia/apex`. ref: https://discuss.pytorch.org/t/torch-cuda-amp-vs-nvidia-apex/74994
2. The installation is duplicated in E3/SM because it's believed to depend on torch to build when the split was done. There should be a separate effort investigating which duplicated sections can be moved to the common stage. This effort is out of the scope of this PR, however.

Reply:
Sagemaker model parallel requires apex, so it should be in the sagemaker dockerfile. Whether should it be in the E3 one depends on the E3 customers if they require apex. Recommendation is to keep it there.